core/src/raw/ops.rs

```diff
@@ -209,6 +209,27 @@ impl From<OpWrite> for PresignOperation {
     }
 }
 
+impl From<OpStat> for OpRead {
```

Comment:
Hi, OpenDAL prioritizes optimization for reading as our code typically only needs to be changed once but may be read by multiple developers. So we will spend more time to write readable and maintainable code instead.

I can see the potential benefit of converting between `OpRead` and `OpStat` since they share most fields. However, allowing convert `OpRead`/`OpStat` makes reader feel confused. Everytime we read a `into()`, we need to figure which type we are using.

So I prefer to remove those changes.